lib/rules/no-curly-component-invocation.js

```diff
@@ -66,6 +101,30 @@ module.exports = class NoCurlyComponentInvocation extends Rule {
     return `You are using the component {{${name}}} with curly component syntax. You should use <${angleBracketName}> instead. If it is actually a helper you must manually add it to the 'no-curly-component-invocation' rule configuration, e.g. \`'no-curly-component-invocation': { allow: ['${name}'] }\`.`;
   }
 
+  fixForBlockNode(node) {
+    if (node.program.type !== 'Block') {
+      return;
+    }
+    if (node.program.chained) {
+      return;
+    }
+    if (node.path.type !== 'PathExpression') {
+      return;
+    }
+
+    if (node.params.length) {
+      return;
+    }
```

Comment:
Since we are returning early in these scenarios, we need to fallback to logNode (because `ember-template-lint . --fix` should still fail when it sees `{{#foo-bar thing}}{{/foo-bar}}`).

Reply:
updated